components/localize/localize-mixin.js

```diff
@@ -13,9 +13,7 @@ export const LocalizeMixin = superclass => class extends superclass {
 	}
 
 	connectedCallback() {
-		if (super.connectedCallback) {
-			super.connectedCallback();
-		}
+		super.connectedCallback();
```

Comment:
Do we have to do this at all if we don't add anything to this function?  Or do we expect we will be adding to it?